app/models/fractie.js

```diff
@@ -1,6 +1,7 @@
 import Model, { attr, belongsTo, hasMany } from '@ember-data/model';
 
 export default class FractieModel extends Model {
+  @attr uri;
```

Comment:
Otherwise Fractie URI is not loaded into frontend

Reply:
this is a bit of a general gotcha :(